src/index.md

```diff
@@ -3,7 +3,7 @@ home: true
 layout: HomeLayout
 background: /assets/hero-bg.jpg
 heroImage: /assets/stratum-v2-icon-with-text.svg
-heroText: A complete, open-source and community-managed implementation of the Stratum V2 protocol. Stratum V2 increases security, makes data transfers more efficient, and reduces mining infrastructure requirements. It also introduces three new sub-protocols that let miners select transaction sets and improve decentralization by negotiating with pools.
+heroText: Open-source and community-managed reference implementation of the Stratum V2 protocol. Stratum V2 increases security, makes data transfers more efficient, and reduces mining infrastructure requirements. It also introduces three new sub-protocols that let miners select transaction sets and improve decentralization by negotiating with pools.
```

Comment:
Looks good but should we keep the article "A" at the beginning? Or rather, it would be "An open-source ... "

Reply:
Done. Thanks.